targets/files/packages/8/p2/drupal/token.json

```diff
@@ -1 +1 @@
-{"packages":{"drupal\/token":[{"keywords":"Actively maintained,Rules","homepage":"https:\/\/www.drupal.org\/project\/token","version":"1.9.0","version_normalized":"1.9.0.0","license":"GPL-2.0-or-later","authors":[{"name":"Berdir","homepage":"https:\/\/www.drupal.org\/user\/214652"},{"name":"Dave Reid","homepage":"https:\/\/www.drupal.org\/user\/53892"},{"name":"eaton","homepage":"https:\/\/www.drupal.org\/user\/16496"},{"name":"fago","homepage":"https:\/\/www.drupal.org\/user\/16747"},{"name":"greggles","homepage":"https:\/\/www.drupal.org\/user\/36762"},{"name":"mikeryan","homepage":"https:\/\/www.drupal.org\/user\/4420"}],"support":{"source":"https:\/\/git.drupalcode.org\/project\/token"},"source":{"type":"git","url":"https:\/\/git.drupalcode.org\/project\/token.git","reference":"8.x-1.9"},"dist":{"type":"zip","url":"https:\/\/ftp.drupal.org\/files\/projects\/token-8.x-1.9.zip","reference":"8.x-1.9","shasum":"a5d234382a1a0e4ba61d4c7a2fa10671ca656be4"},"type":"drupal-module","uid":"token-3189015","name":"drupal\/token","extra":{"drupal":{"version":"8.x-1.9","datestamp":"1608284866","security-coverage":{"status":"covered","message":"Covered by Drupal\u0027s security advisory policy"}},"drush":{"services":{"drush.services.yml":"^9 || ^10"}}},"description":"Provides a user interface for the Token API, some missing core tokens.","require":{"drupal\/core":"^8.8 || ^9"}}]},"minified":"composer\/2.0"}
```

Comment:
Oh, I remember. I removed the dependency on `drupal/core`, since that made the test interact with the internet and was not needed for our purposes anyway.